### ISSUE TITLE
Recurring transactions starting from the past

### ISSUE THREAD

**Issue by crazyyzarc** [firefly-iii/firefly-iii#8239]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)
- [X] My request is not listed as [a very good idea, but unfortunately...](https://docs.firefly-iii.org/firefly-iii/more-information/what-its-not/)
- [X] I've used [the search](https://github.com/firefly-iii/firefly-iii/issues?q=is%3Aissue) and this has not been requested before.

### Description

How can I add old transactions from the beginning of 2023 with recurring transactions?
I want to add all automated transactions from 2023 with recurring automations, but I can't be starting from 01/01/2023.

![2023-12-07_19-30-22-910_7UTU7](https://github.com/firefly-iii/firefly-iii/assets/4954535/71a0f3fd-0ac2-4e42-a930-abe567e37de0)

### Solution

Add automations also from the past to the future

**Comment by JC5** [comment]
You can't. Recurring transactions can create future transactions only. They can't be applied retroactively.

**Comment by crazyyzarc** [comment]
> You can't. Recurring transactions can create future transactions only. They can't be applied retroactively.

Doesn't it make sense to introduce this function? I mean... if you always check your finances afterwards (i.e. at the end of the year or at the end of the month) then some standing orders are missing.

**Comment by JC5** [comment]
Not really, no. A recurring transaction prevents this. Just not retroactively.

**Comment by crazyyzarc** [comment]
> Not really, no. A recurring transaction prevents this. Just not retroactively.

And how do you manage to quickly record recurring transactions in the past retrospectively? Adding them manually is not a quick solution

**Comment by JC5** [comment]
Use the clone and edit button. All you need to change is the date.
